modules/codemod/lib/v8/spec/renameLoadingAnimation.spec.ts

```diff
@@ -0,0 +1,119 @@
+    });
+
+    it('should update renamed exports', () => {
+      const input = stripIndent`
+        import {LoadingAnimation as CanvasLoadingAnimation} from "@workday/canvas-kit-react/loading-animation";
+      `;
+
+      const expected = stripIndent`
+        import {LoadingDots as CanvasLoadingAnimation} from "@workday/canvas-kit-react/loading-dots";
+      `;
+
+      expectTransform(input, expected);
+    });
+
+    it('should update slash import with LoadingDots', () => {
+      const input = stripIndent`
+        import {LoadingDots} from "@workday/canvas-kit-react/loading-animation";
+      `;
+
+      const expected = stripIndent`
+        import {LoadingDots} from "@workday/canvas-kit-react/loading-dots";
+      `;
+
+      expectTransform(input, expected);
+    });
+  });
+
+  describe('when updating usages', () => {
+    it('should update JSX usage', () => {
+      const input = stripIndent`
+        import {LoadingAnimation} from "@workday/canvas-kit-react/loading-animation";
+
+        <LoadingAnimation />
+      `;
+
+      const expected = stripIndent`
+        import {LoadingDots} from "@workday/canvas-kit-react/loading-dots";
+
+        <LoadingDots />
+      `;
+
+      expectTransform(input, expected);
+    });
+
+    it('should update JSX with attributes unaffected', () => {
+      const input = stripIndent`
+        import {LoadingAnimation} from "@workday/canvas-kit-react/loading-animation";
+
+        <LoadingAnimation className='loading-centered' data-some-prop="untouched" />
+      `;
+
+      const expected = stripIndent`
+        import {LoadingDots} from "@workday/canvas-kit-react/loading-dots";
+
+        <LoadingDots className='loading-centered' data-some-prop="untouched" />
+      `;
+
+      expectTransform(input, expected);
+    });
+
+    it('should update styled components', () => {
+      const input = stripIndent`
+        import {LoadingAnimation} from "@workday/canvas-kit-react/loading-animation";
+
+        const StyledLoadingAnimation = styled(LoadingAnimation)({ width: space.xxs, height: space.xxs });
+      `;
+
+      const expected = stripIndent`
+        import {LoadingDots} from "@workday/canvas-kit-react/loading-dots";
+
+        const StyledLoadingAnimation = styled(LoadingDots)({ width: space.xxs, height: space.xxs });
+      `;
+
+      expectTransform(input, expected);
+    });
+    it('should not update non-Canvas styled components', () => {
+      const input = stripIndent`
+        import {LoadingAnimation as CanvasLoadingAnimation} from "@workday/canvas-kit-react/loading-animation";
+
+        const StyledLoadingAnimation = styled(LoadingAnimation)({ width: space.xxs, height: space.xxs });
+      `;
+
+      const expected = stripIndent`
+        import {LoadingDots as CanvasLoadingAnimation} from "@workday/canvas-kit-react/loading-dots";
+
+        const StyledLoadingAnimation = styled(LoadingAnimation)({ width: space.xxs, height: space.xxs });
+      `;
```

Comment:
Sorry this is confusing, and please suggest how this could be different: "should not update non-Canvas styled components".

This was based on an usage I found and I wanted to make sure that Identfiers that match "LoadingAnimation" do not get updated even when Canvas is used. So it should NOT be `CanvasLoadingAnimation`